src/Engine/ProtoCore/Core.cs

```diff
@@ -54,6 +54,10 @@ public class Options
     {
         public Options()
         {
+            // Execute using new graphnode dependency
+            DirectDependencyExecution = true;
+            LHSGraphNodeUpdate = !DirectDependencyExecution;
```

Comment:
Is `LHSGrpahNodeUpdate` always is `!DirectDependencyExecution`? If it is, then we may remove `LHSGraphNodeUpdate`.